go.mod

```diff
@@ -0,0 +1,19 @@
+module smod
```

Comment:
what was the reason that you kept the module name as a `smod`?
Ideally, you should keep your module name as the GitHub repo path. In this case, the module name should be: `github.com/ppsingh4/stu-mini`